server/monitor/src/main/java/org/apache/accumulo/monitor/rest/tables/TableInformationList.java

```diff
@@ -21,15 +21,15 @@
 
 /**
  *
- * Generates a list with table information
+ * Generates a list with table information. This is mainly used to populate DataTables
```

Comment:
Good point.  I put this comment to explain the changes in the class below but perhaps I should just comment next to the "data" field as to why its called something as generic as data.  I tried to get DataTables to work with the JSON object as  "table" but couldn't and it wasn't pretty.

Reply:
FYI I figured out how to tell DataTables to look for "table" vs "data".  Its easy I was just using the wrong json parameter.  This will make it so less changes are needed and we can use the POJOs already returned by our rest endpoints.